modules/salt/pages/custom-states.adoc

```diff
@@ -0,0 +1,151 @@
+[[custom-states]]
+= Custom Salt States
+
+You can create your own custom Salt states with {productname} as centrally managed configuration channels.
+Custom states are stored as Salt state files on the {productname} Server with a ``.sls`` extension, in the [path]``/srv/susemanager/salt/`` directory.
```

Comment:
`/srv/susemanager/salt/` has content which is managed by SUMA. I think no customer should mess around with files in this directory. So I am not sure if it is wise to tell them where it is. At least not at such a prominent place?